apple/Handlers/RNPanHandler.m

```diff
@@ -80,6 +88,31 @@ - (void)setMinimumNumberOfTouches:(NSUInteger)minimumNumberOfTouches
 }
 #endif
 
+#if !TARGET_OS_OSX && !TARGET_OS_TV
+- (void)tryUpdateStylusData:(UIEvent *)event
+{
+  UITouch *touch = [[event allTouches] anyObject];
+
+  if (touch.type != UITouchTypePencil) {
+    return;
+  }
+
+  _stylusData.altitudeAngle = touch.altitudeAngle;
+  _stylusData.azimuthAngle = [touch azimuthAngleInView:nil];
+  _stylusData.pressure = touch.force / touch.maximumPossibleForce;
+
+  RNGHTilt tilts = ghSpherical2tilt(_stylusData.altitudeAngle, _stylusData.azimuthAngle);
+
+  _stylusData.tiltX = tilts.tiltX;
+  _stylusData.tiltY = tilts.tiltY;
+}
+
+- (RNGHStylusData *)getStylusData
+{
+  return _stylusData;
+}
```

Comment:
```suggestion
```

Reply:
Done in [b2a7437](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/b2a7437d18c51829dc29baf7cc62c65b3b419d9c)